### PR TITLE
mac/xplat intro to mvc, update to 2.0

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Review URLs
* [Mac](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app-mac/adding-model?branch=pr-en-us-4131#prepare-the-project-for-scaffolding)
* [VS Code](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app-xplat/adding-model?branch=pr-en-us-4131#prepare-the-project-for-scaffolding)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@scottaddie  hold off, more commit on the way

**Comment by Rick-Anderson** [comment]
@scottaddie  ready for review.

File: aspnetcore/includes/mvc-intro/adding-model2xp.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## Perform initial migration
+
+From the command line, run the following .NET Core CLI commands:
+
+```console
+dotnet ef migrations add InitialCreate
+dotnet ef database update
+```
+
+The `ef migrations add InitialCreate` command generates code to create the initial database schema. The schema is based on the model specified in the `DbContext` (In the *Models/MovieContext.cs* file). The `Initial` argument is used to name the migrations. You can use any name, but by convention you choose a name that describes the migration. See [Introduction to migrations](xref:data/ef-mvc/migrations#introduction-to-migrations) for more information.
```
Comment (scottaddie) [inline aspnetcore/includes/mvc-intro/adding-model2xp.md]: Should this command be prefixed with "dotnet"?

File: aspnetcore/includes/mvc-intro/adding-model2xp.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+The `ef database update` command runs the `Up` method in the *Migrations/\<time-stamp>_InitialCreate.cs* file, which creates the database.
```
Comment (scottaddie) [inline aspnetcore/includes/mvc-intro/adding-model2xp.md]: Should this command be prefixed with "dotnet"?

**Pull request merged**